docker/dockerfiles/dev-env/dev-env.Dockerfile

```diff
@@ -54,6 +54,11 @@ FROM starrocks/starlet-artifacts-ubuntu22:${starlet_tag} as starlet-ubuntu
 FROM starrocks/starlet-artifacts-centos7:${starlet_tag} as starlet-centos7
 # determine which artifacts to use
 FROM starlet-${distro} as starlet
+# remove unnecessary and big starlet dependencies
+COPY --from=builder /root/starrocks/docker/dockerfiles/dev-env/starlet_exclude .
+RUN while read line; do \
```

Comment:
RUN `cd release && cat starlet_exclude | xargs -r rm -rvf`

Reply:
possibly exclude lines start with `#` so it can be easily unmask the delete line.